cloudformation/aws-parallelcluster.cfn.json

```diff
@@ -1561,6 +1584,16 @@
                   "Fn::Sub": "arn:${AWS::Partition}:iam::${AWS::AccountId}:role/parallelcluster-*"
                 }
               ]
+            },
+            {
+              "Sid": "DcvLicense",
+              "Effect": "Allow",
+              "Action": "s3:GetObject",
+              "Resource": [
+                {
+                  "Fn::Sub": "arn:aws:s3:::dcv-license.${AWS::Region}/*"
```

Comment:
What about other partitions such as `aws-cn` and `aws-gov`? see https://docs.aws.amazon.com/govcloud-us/latest/UserGuide/using-govcloud-arns.html#using-govcloud-arn-format

Reply:
Fixed